### ISSUE TITLE
Daily meeting ( 14/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Gallery-Shopping-Website#3]
So how the progress guys?

**Comment by Lai0723** [comment]
For me i just finish with the cart database and today i currently doing the role database to store the admin role that can update and delete the content of the website.

**Comment by kcchan3012** [comment]
For me i just finished with the design of the menu page and the product page and currently i link the page with database to retrieve the data and images from the database.

**Comment by boonsenglim** [comment]
I just finish with the validation for the registration part and now i link to database to store the user data to the user database.

**Comment by pqyyieeuyi** [comment]
While i just finished with everything for the login design and now I need boon seng to store the data to database so that i can validate the user login.

**Comment by Lai0723** [comment]
By the way, kah chen do you create a master page ?

**Comment by kcchan3012** [comment]
No, why we need it.

**Comment by pqyyieeuyi** [comment]
So that i we can use the same design for every page that link to the master page.

**Comment by boonsenglim** [comment]
Ya, benjamin is right this will give us convenient because we just need to change the design on the master page instead of change every page.

**Comment by Lai0723** [comment]
I agree with boon seng and benjamin , we better link all the page with the master page.

**Comment by kcchan3012** [comment]
ok, i will change it after i successful link retrieve data from the database. I think nothing to report today anymore.

**Comment by Lai0723** [comment]
Ok, so everyone can dismiss now and continue with the work on the project.
